vsintegration/packages.config

```diff
@@ -18,6 +18,8 @@
   <package id="Microsoft.VisualStudio.Shell.Design" version="14.3.25407" targetFramework="net46" />
   <package id="Microsoft.VisualStudio.Settings.15.0" version="15.0.25824-RC" targetFramework="net46" />
   <package id="Microsoft.VisualStudio.Utilities" version="14.3.25407" targetFramework="net46" />
+  <package id="Microsoft.VisualStudio.Imaging" version="14.3.25407" targetFramework="net46" />
```

Comment:
Should you use:
https://www.nuget.org/packages/Microsoft.VisualStudio.Imaging/15.0.25901-RC
and
https://www.nuget.org/packages/Microsoft.VisualStudio.ImageCatalog/15.0.25824-RC